docs/source/configuration/index.rst

```diff
@@ -19,6 +19,9 @@ Due to this fine-grained control, there are a very large number of settings.
 **It is not necessary to configure everything.**
 pseudopeople includes reasonable default noise settings
 and your configuration can override as few or as many of the default values as you like.
+You can also pass the special value :code:`pseudopeople.NO_NOISE`, which prevents all configurable noise types
+from occurring at all.
+
```

Comment:
I don't think we directly considered that; it's not a bad idea but it does add an extra "degree of freedom" i.e. someone could set the boolean to False but pass a _configuration_ that specifies no noise, or vice versa.

The other reason I like this API is that in the future, `pseudopeople.NO_NOISE` could change from being a sentinel value to actually being the full configuration for no noise, and it would be backwards-compatible.